### ISSUE TITLE
Can the support for TSX be the same as JSX ?

### ISSUE THREAD

**Issue by angelporo** [smihica/emmet-mode#136]
As the title

**Comment by zhyuanyuan** [comment]
yes,I find this problem in typescript-mode to edit TSX, for exmple div.say -> <div class='say'></div>. isn't <div className="say"></div>. How to set like js2-mode.

**Comment by zhyuanyuan** [comment]
oh. I find how to solve this problem. this is my configure.

;;; zencoding
(use-package emmet-mode
  :ensure t
  :defer 2
  :commands emmet-mode
  :hook
  ((web-mode css-mode sgml-mode rjsx-mode typescript-mode js2-mode) . emmet-mode)
  :init
  (setq emmet-indent-after-insert nil)
  (setq emmet-indentation 2)
  (setq emmet-expand-jsx-className? t)  ; 设置jsx 语法
  (setq emmet-move-cursor-between-quotes t)
  (setq emmet-self-closing-tag-style " /")

  :config
  (add-to-list 'emmet-jsx-major-modes 'js-mode)
  (add-to-list 'emmet-jsx-major-modes 'typescript-mode)
  ;; (setq emmet-jsx-className-braces? t) ; className={}
  )
